src/components/App.js

```diff
@@ -1,25 +1,22 @@
-import logo from '../assets/logo.svg';
 import '../styles/App.css';
+import '../styles/index.css';
+import {BrowserRouter, Route, Routes, Router} from "react-router-dom";
+import SignUp from './SignUp';
+import SignIn from './SignIn';
+import Error from './Error';
 
 function App() {
+
   return (
-    <div className='App'>
-      <header className='App-header'>
-        <img src={logo} className='App-logo' alt='logo' />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className='App-link'
-          href='https://reactjs.org'
-          target='_blank'
-          rel='noopener noreferrer'
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <BrowserRouter>
+  
+      <Routes>
```

Comment:
`Routes`? I couldn't find anything online about this component from `react-router-dom`.

Reply:
Routes is in react-router-dom. Switch is now unavailable in the new version of react-router-dom. I tried Switch first but it told me that it was not found in react-router-dom so I chose Routes to finish the routing. We can use switch but we will have to use an older version of react-router-dom.

Reply:
I didn't know that, thanks. As long as we're using a stable version of react-router-dom, there is no issue.